config/enterprise_versions.yml

```
@@ -68,3 +68,6 @@ components:
   guardian:
     image: tigera/guardian
     version: v2.7.0-0.dev-61-g73e2b02
+  cni:
```

The key of this yaml entry is important. 
- It needs to match what calico-private and tigera/process are using.
- It also should ideally be unique so that the key for the OS component and the Private variant of that same component don't use the same key. (this is a new rule because if they are identical they can't be put in the same versions.yml file which is annoying).


How about this intead?
```suggestion
  tigera-cni:
```

---

Sounds good!